README.md

```diff
@@ -3,3 +3,20 @@
 The public website for the FLIF project.
 
 * **[FLIF.info](http://flif.info)**
+
+* * *
+
+## Contributing
+
+To run the app locally:
```

Comment:
"app"?

This is only needed for the fancy 404 stuff, if you just want to modify the content you can edit and view the site locally. No need to scare away potential contributors.